HHS/OPRE-OPS
# Updated Agreement List table with Next Budget Line

## What changed

* new/updated Agreements list columns with Next Budget Line 
* logic to find Next Budget Line using only PLANNED and IN_EXECUTION status

## Issue

#1655 #965 

## How to test

View the Agreements List to see the new Next Budget Line and update Next Need By columns

## Screenshots

![image](https://github.com/HHS/OPRE-OPS/assets/127965976/f32ca042-28c0-4adf-a4d0-5d1abde80e9d)

## Definition of Done Checklist


## Links

_If relevant, e.g. for a link to a piece of markdown documentation_
